### PR TITLE
feat:replace titre logo img

### LINKED ISSUE #1: Remplacer le titre Aixmazone par le logo
:bulb: Utiliser le [composant Image](https://nextjs.org/docs/api-reference/next/image#required-props) de Next.

:point_right: Ça se passe par ici : 
https://github.com/les-petits-commercants-de-la-toile/aixmazon-v1-2022-PoolA/blob/f4ec8cb6ff916dcd3b15a60ce2a38c739e4d2ae4/storefront/features/Header/Header.tsx#L8

aixmazone-logo

### PR DESCRIPTION
**Pull request by manoncayetano**
Close #1

### CODE REVIEW

File: storefront/features/Header/Header.tsx [hunk 1]
```diff
@@ -1,11 +1,18 @@
...
+import Image from 'next/image'
+import logo from '../images/logo.png'
 
 const Header = () => {
   return (
     <Wrapper>
-      <Heading textAlign="center">Aixmazone</Heading>
+      <Heading textAlign="center">
+      <Image
+              src={logo}
```
Comment (shouze) [inline storefront/features/Header/Header.tsx:12]: Regarding accessibility, can't you add something that could help blind people that can't see images for example?

File: storefront/features/Header/Header.tsx [hunk 2]
```diff
@@ -1,11 +1,18 @@
 import styled from "styled-components";
 import { Heading } from "../../components";
 import Navbar from "../../containers/Navbar/Navbar";
+import Image from 'next/image'
+import logo from '../images/logo.png'
```
Comment (shouze) [inline storefront/features/Header/Header.tsx:5]: :+1:

**Review by shouze — APPROVED** [review]
Feel free to make another pull request to improve accessibility on this logo image

**Review by manoncayetano — COMMENTED** [review]
Add alt for the logo

**Pull request merged**